frontend/src/pages/projects/screens/detail/storage/useProjectPvcs.ts

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { getPvcs } from '../../../../../api';
+import { PersistentVolumeClaimKind } from '../../../../../k8sTypes';
+
+const useProjectPvcs = (
+  namespace?: string,
+): [
+  pvcs: PersistentVolumeClaimKind[],
+  loaded: boolean,
+  loadError: Error | undefined,
+  forceRefresh: () => void,
+] => {
+  const [pvcs, setPvcs] = React.useState<PersistentVolumeClaimKind[]>([]);
+  const [loaded, setLoaded] = React.useState(true);
+  const [loadError, setLoadError] = React.useState<Error | undefined>(undefined);
+
+  const getProjectPvcs = React.useCallback(() => {
+    if (namespace) {
+      setLoaded(false);
```

Comment:
You don't want to do this -- let it update organically -- once we are loaded, stay loaded.